### ISSUE TITLE
Update code.json to 2.0 spec/schema

### ISSUE THREAD

**Issue by pmpowers-usgs** [usgs/best-practices#58]
It appears that this is the metric by which agencies will be judged compliant wrt software. In recently bringing my repos up to spec, it seems the code.json specification has been updated, somewhat significantly, to 2.0, with documentation of the prior spec/schema removed. The [current file](https://github.com/usgs/best-practices/blob/master/code.json) in this repo, which is cited as an example, will lead users astray. See the [nshmp-haz `code.json`](https://github.com/usgs/nshmp-haz/blob/master/code.json) for a compliant example.

https://code.gov/#/policy-guide/docs/compliance/inventory-code

**Comment by emartinez-usgs** [comment]
Yes - There is a PR in place to update the reference documentation in this repo.

https://github.com/usgs/best-practices/pull/54
